study-builder/tenants/cmi/change_password_email.html

```diff
@@ -0,0 +1,140 @@
+<html>
```

Comment:
One of the things we have to update for a new study is this "Change Password" email in Auth0 but we didn't have this checked in, so putting it in now. (This can be found in Auth0 dashboard > Emails > Templates > Change Password in dropdown.)